### PR TITLE
MAVLINK Bridge

### PR DESCRIPTION
**Pull request by njoubert**
This adds a UDP client that sends only observations and base position messages over UDP.

### CODE REVIEW

**Comment by fnoble** [comment]
Looks good but I wonder if this is the best place for this to live. Not sure where would be better, maybe `piksi_tools`?

File: python/sbp/client/examples/mavlinkbridge.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                      help="specify the baud rate to use.")
+  parser.add_argument("-a", "--address",
+                      default=[DEFAULT_UDP_ADDRESS], nargs=1,
+                      help="specify the serial port to use.")
+  parser.add_argument("-p", "--udp-port",
+                      default=[DEFAULT_UDP_PORT], nargs=1,
+                      help="specify the baud rate to use.")
+  return parser.parse_args()
+
+def send_udp_callback_generator(udp, args):
```
Comment (mookerji) [inline python/sbp/client/examples/mavlinkbridge.py]: I believe there's is a packing function that does this for you in the library.

**Comment by mookerji** [comment]
Sweet! I echo @fnoble 's suggestion. This should probably be in piksi_tools. If you could also add a test of some kind, that would be great.

File: python/sbp/client/examples/mavlinkbridge.py (hunk 1 above)
Comment (njoubert) [inline python/sbp/client/examples/mavlinkbridge.py]: This is just a copy of Fergus' original code, can you link me to an example of what you're referring to?

Comment (mookerji) [inline python/sbp/client/examples/mavlinkbridge.py]: Yeah, see: 
https://github.com/swift-nav/libsbp/blob/master/python/sbp/__init__.py#L103

**Comment by mookerji** [comment]
Closed in favor of https://github.com/swift-nav/piksi_tools/pull/74/.

**Pull request closed without merging**